.travis.yml

```diff
@@ -1,18 +1,32 @@
-sudo: false
+sudo: true
```

Comment:
Installing repos to sources list or installing deb's requires sudo. We can revert back once this is fixed: https://github.com/travis-ci/apt-source-whitelist/issues/256